internal/events/trigger/trigger_event.go

```diff
@@ -118,7 +118,7 @@ func Fire(p TriggerParameters) (string, error) {
 			JSON:           resp.JSON,
 			Secret:         p.Secret,
 			ForwardAddress: p.ForwardAddress,
-			Event:          e.GetTopic(p.Transport, p.Event),
+			Event:          p.Event,
```

Comment:
Removing this has a byproduct; the CLI also supports shorthand aliases for most of the events, such as `drop` or `add-redemption`. 

By bypassing that here, while it fixes your specific error, it breaks those users as it uses this to get the Topic. 

The fix here is to add a safety check here to check if `GetTopic` is an empty string (invalid shorthand) and then get the `GetEventSubAlias`  to see if it returns a value (which returns the shorthand). Basically: 

```go
		topic := e.GetTopic(p.Transport, p.Event)
		if topic == "" && e.GetEventSubAlias(p.Event) != "" {
			topic = p.Event 
		}
```

Reply:
I didn't know about shorthands 😄 fixed it.